### ISSUE TITLE
Empty echo command in add-on does not print empty line

### ISSUE THREAD

**Issue by hferentschik** [minishift/minishift#1064]
Adding an empty "echo" command in an add-on to print an empty lines fails. Should just print the empty line.

**Comment by LalatenduMohanty** [comment]
@hferentschik It is not failing for me. However there seems to be a bug for sure. For example if I execute an addon with following content
```
# Name: foo
# Description: Changes the default security context constraints to allow pods to run as any user.
# Url: https://docs.openshift.org/latest/creating_images/guidelines.html#openshift-origin-specific-guidelines

echo  hello
echo
echo  bye
~         
```
The out put contains the `echo` as below. 
```
-- Applying addon 'foo':
   hello
  echo
   bye
```

**Comment by budhrg** [comment]
This is happening in this PR as well https://github.com/minishift/minishift/pull/1043

**Comment by hferentschik** [comment]
@LalatenduMohanty, so it is "failing", right?

**Comment by hferentschik** [comment]
Merged via pull request #1079
